.github/workflows/Release.yml

```diff
@@ -138,6 +138,9 @@ jobs:
         with:
           gradle-home-cache-cleanup: true
 
+      - name: Setup .npmrc
+        run: echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" > .npmrc
```

Comment:
I think, you could use `${{ secrets.NPM_TOKEN }}`

Reply:
Probably, but NPM will do its own substitution anyway so I don't think it really matters.